### PR TITLE
Add google tracking to Sphynx docs

### PR DESCRIPTION
**Pull request by kerinin**

### CODE REVIEW

File: python/docs/source/_templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends "!layout.html" %}
```
Comment (bjchambers) [inline python/docs/source/_templates/layout.html:1]: Should change the template -- the theme already supports this.

File: python/docs/source/conf.py [hunk 2]
```diff
@@ -69,6 +69,7 @@
     "github_repo": "kaskada",
     "github_version": "main",
     "doc_path": "kaskada/docs/source",
+    "google_analytics": "G-HR9E2E6TG4",
```
Comment (bjchambers) [inline python/docs/source/conf.py:72]: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/analytics.html

```
html_theme_options["analytics"] = {
    "google_analytics_id": "G-XXXXXXXXXX",
}
```

OR

```
html_theme_options = {
  ...
  "analytics": {
    "google_analytics_id": "...",
  }
}
```

Comment (bjchambers) [inline python/docs/source/conf.py:72]: We may also want this to be an environment variable, so that it is only on in the deployed docs?

Comment (kerinin) [reply python/docs/source/conf.py]: Let's hard code it for now - it's what we're doing in a couple other places.

**Review by bjchambers — APPROVED** [review]

**Pull request merged**